### DISCUSSION TITLE
Per-component translations seem to only work with Composition API?

### DISCUSSION THREAD

**Discussion by RodrigoProjects** [nuxt-modules/i18n#2167] [General]
Hello everyone! :)

I oppened this discussion because I would like to understand why it seems i18n is moving a part from Options API and going for a more Composition API approach. Maybe it's just because the Options API docs are less than Composition API an I cannot find how to use per-component translations without the `usei18n` composable! 

Thanks in advance for intereacting with this discussion!

**Comment by TheDutchCoder** [comment]
I think they just don't support it, which is a real problem fr people migrating from Nuxt 2 or 3 (or people preferring the Options API in general).
